frontend/js/src/forms/DeleteApplication.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { JSX } from "react";
+import { createRoot } from "react-dom/client";
+import { getPageProps } from "../utils";
+
+type DeleteApplicationProps = {
+  csrf_token: string;
+  cancel_url: string;
+  application: Application;
+};
+
+function DeleteApplication({
+  csrf_token,
+  cancel_url,
+  application,
+}: DeleteApplicationProps): JSX.Element {
+  return (
+    <>
+      <h2>Delete application</h2>
+      <hr />
+      <div className="card">
+        <div className="card-body">
+          <h5 className="card-title">Client Details</h5>
+          <p className="card-text">
+            <strong>Name:</strong> {application.name}
+          </p>
+          <p className="card-text">
+            <strong>Description:</strong> {application.description}
+          </p>
+          <p className="card-text">
+            <strong>Website:</strong>{" "}
+            <a href={`${application.website}`}>{application.website}</a>
+          </p>
+          <p>Are you sure you want to delete this client?</p>
+          <form method="POST" className="form-horizontal">
+            <div className="form-group">
+              <div className="col-sm-offset-4 col-sm-5">
+                <input
+                  id="csrf_token"
+                  name="csrf_token"
+                  type="hidden"
+                  value={csrf_token}
+                />
+                <input id="confirm" name="confirm" type="hidden" value="yes" />
+              </div>
+            </div>
+            <div className="form-group">
+              <div className="col-md-offset-3 col-md-1">
+                <a href={cancel_url} className="btn btn-default">
+                  Cancel
+                </a>
+              </div>
+              <div className="col-md-1" style={{ marginLeft: "8px" }}>
+                <button type="submit" className="btn btn-danger">
+                  Delete
+                </button>
+              </div>
+            </div>
```

Comment:
On smaller sizes the button stack up weirdly.
![image](https://github.com/user-attachments/assets/5fb1e8c2-5397-4a02-a25b-43a15128f1d6)

You can use a bootstrap btn-group instead of using col-* utilities like so:

```suggestion
            <div className="btn-group">
              <a href={cancel_url} className="btn btn-default">
                Cancel
              </a>
              <button type="submit" className="btn btn-danger" style={{ marginLeft: "8px" }}>
                Delete
              </button>
            </div>
```



Result 
![image](https://github.com/user-attachments/assets/bf313109-ac8a-403c-a93f-859b8dbdadcc)